core/bot_framework_adapter.go

```diff
@@ -84,6 +85,22 @@ func NewBotAdapter(settings AdapterSetting) (Adapter, error) {
 	return &BotFrameworkAdapter{settings, auth.NewJwtTokenValidator(), connectorClient}, nil
 }
 
+// GetSenderInfo returns information about message sender.
+func (bf *BotFrameworkAdapter) GetSenderInfo(ctx context.Context, req schema.Activity) (*schema.ConversationMember, error) {
+	response, err := activity.NewActivityResponse(bf.Client)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to create response object.")
+	}
+
+	resp, err := response.GetSenderInfo(req)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to get data.")
+	}
+
+	return resp, nil
+
```

Comment:
```suggestion
	return resp, nil
```
[nit] unnecessary new lines